### PR TITLE
Js02 week02 homework

### PR DESCRIPTION
**Pull request by Laurelinduilwen**

### CODE REVIEW

File: Week2/homework/maartjes-work.js [hunk 1]
```diff
@@ -42,20 +42,27 @@ const tuesday = [
...
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const sumEarnings = tasks
+    .map(task => task.duration / 60) // mapping the duration values in seconds to hours
+    .filter(timeElapsed => timeElapsed >= 2) // eliminating stuff that took less than 2 hours
+    .reduce((income, timeElapsed) => income + timeElapsed * hourlyRate, 0); // ending the chain with a nice reduce
+  return sumEarnings;
```
Comment (yash-kapila) [inline Week2/homework/maartjes-work.js]: 👍

File: Week2/homework/map-filter.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
-function doubleOddNumbers(numbers) {
-  // Replace this comment and the next line with your code
-  console.log(numbers);
-}
+const doubleOddNumbers = numbers => {
+  let newNumbers = [];
+  newNumbers = numbers.filter(number => number % 2 !== 0);
+  newNumbers = newNumbers.map(x => x * 2);
+  return newNumbers;
+};
```
Comment (yash-kapila) [inline Week2/homework/map-filter.js]: Ah - you can also chain the two methods `filter` and `map` together and rewrite this code as: 

```
const newNumbers = numbers.filter(number => number % 2 !== 0).map(x => x * 2);
return newNumbers;
```

**Comment by Laurelinduilwen** [comment]
thank you for the feedback Yash :D

File: Week2/homework/map-filter.js (hunk 2 above)
Comment (Laurelinduilwen) [reply Week2/homework/map-filter.js]: ahh yeah, I kinda forgot updating it since I did this one first and learned it later 😅 
will update ^^

**Pull request closed without merging**